docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,125 @@
+---
+product: material-ui
+title: React Modal component
+components: Modal, ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other windows that force the user to take action before continuing.</p>
+
+The `Modal` component renders its `children` node in front of a backdrop component. This lets you create a window that your users must interact with before continuing in the parent application.
+Features:
+
+- 💄 Manages modal stacking when more than one is needed
+- 🔐 Creates a backdrop to disable interaction with the rest of the app
+- 🔐 Disables page scrolling while open
+- ♿️ Manages focus correctly between the modal and its parent app
+- ♿️ Adds the appropriate ARIA roles automatically
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+> **Note:** the term "modal" is sometimes used interchangeably with "dialog," but this is incorrect.
+> A dialog may be _modal_ or _nonmodal_.
+> 
+> A modal [blocks interaction with the rest of the application](https://en.wikipedia.org/wiki/Modal_window), forcing the user to take action.
+> As such, it should be used sparingly—only when the app _requires_ user input before it can continue.
+
+If you are creating a modal dialog, the [`Dialog`](/material-ui/dialog/) component is better suited for this specific use case.
```

Comment:
This means then that I'm not able to create a Dialog from this Base Modal component?

What if I wanted to create my own version of the below-mentioned components? Imagine I don't want anything to do with Material. How would I proceed? Should we cover this somehow?

Reply:
The impression I got from the original wording is that it is _possible_ but not _preferable_. But I could be wrong about that.

Reply:
The original intention was: If you are creating a modal dialog, the Dialog is better suited for that use-case, as basically it cannot be closed without taking any action. You can achieve the same with a modal, but it will require more work. The modal is more of a utility component used to create different components, like dialogs, popovers, drawers etc.

Reply:
I think it's good that we clarify this further because I could not want to install @mui/material because of one or two components. Maybe it's worth the "more work" instead of adding an additional library to my project (trying to think with a developer's mind here — I've seen this reasoning multiple times). 

So, adding examples on _how_ to create a dialog, popovers, drawers, based on it might be a very good & handy thing.

Reply:
Great idea @danilo-leal, this is a good opportunity to "show" instead of "tell." So when we say "the Dialog is better for this use case," we can point directly to code samples that show how and why.

Reply:
We are going to create these components in the base package, we just haven't done that yet. Once we have them, we can simply link them to the users. How about for now, we remove this sentence and add it back once we have an unstyled dialog?

Reply:
I've commented the line and added comment to add it back once we have an unstyled dialog.